### DISCUSSION TITLE
Encryption in a cluster deployment

### DISCUSSION THREAD

**Discussion by ymesika** [orientechnologies/orientdb#9438] [General]
There is a [note at the bottom of the 3.0.x documentation](http://orientdb.com/docs/3.0.x/security/Database-Encryption.html#opening-encrypted-databases) about encryption no longer supported for cluster deployments.
Out of curiousity, why?

**Comment by andrii0lomakin** [comment]
Hi. That is because we support encryption on file level. So all data are encrypted including indexes and WAL, you merely do not need to encrypt clusters anymore.
